packages/isar_inspector/windows/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+# Project-level configuration.
```

Comment:
Is any of these windows specific files modified? If no, I'd prefer not to check them in.